src/FirstSteps/Program.cs

```diff
@@ -8,33 +8,25 @@ class Program
     {
         static void Main(string[] args)
         {
-
+            var user = Console.ReadLine();
             AnsiConsole.MarkupLine($"{Emoji.Known.BowAndArrow} [darkgreen]Welcome to the RPG game [/] {Emoji.Known.CrossedSwords}");
 
             Game.CreateHero();
 
             AnsiConsole.MarkupLine($"{Emoji.Known.Dagger} [red]Let's play the game [/] {Emoji.Known.Dagger}");
 
             Console.WriteLine("Type 'end' if you want to quit.");
-            Console.WriteLine("\nIf You want go back to the previous menu press 'x'\n");
+            Console.WriteLine("\nIf You want go back to the previous menu press button\n");
             Console.WriteLine("________________________________");
-            string command = AnsiConsole.Prompt(
-            new SelectionPrompt<string>().Title("\nSelect what you want to do: ").AddChoices("stats", "treasure", "inventory", "forest", "dungeons", "mine"));
-
-            Console.WriteLine($"\nYou choose: { command}");
-            Game.HandleCommand(command);         
             do
```

Comment:
Example that should help you:
```
string userInput;
do
{
    string fruit = AnsiConsole.Prompt(
        new SelectionPrompt<string>()
            .Title("Choose your favorite fruit")
            .AddChoices("banana", "apple", "raspberry", "orange"));

    DoSomethingWithFruit(fruit);

    AnsiConsole.MarkupLine("Type 'exit' if you want to shut down the app. Otherwise, you will continue.");
    //here you have to take a text from the user and save it in userInput variable
}
while ();//here you have to check if userInput variable is not equal 'exit'
```